test/protractor.conf.js

```diff
@@ -92,6 +92,7 @@ exports.config = {
     'add-template-to-project': 'integration/features/user_adds_template_to_project.spec.js',
     'add-imagestream-to-project': 'integration/features/user_adds_imagestream_to_project.spec.js',
     'create-from-url': 'integration/features/user_creates_from_url.spec.js',
+    'key-value-unit-tests': 'spec/directives/env-var-test-spec.js'
```

Comment:
You won't need to add anything to protractor for unit tests, this file is for running e2e tests (in the browser).